src/components/NavBar/NavbarDesktop.jsx

```diff
@@ -61,9 +60,7 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+        <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo'aria-label="logo"/>
```

Comment:
see other comment, but do styling inline for now. we are trying to get rid of `styles.css` file.

Reply:
sorry i just seen this now. I will get do inline styling.